app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb

```diff
@@ -6,7 +6,9 @@ def create_in_provider(manager_id, params)
       params = provider_params(params) if respond_to?(:provider_params)
       manager = ExtManagementSystem.find(manager_id)
       tower_object = provider_collection(manager).create!(params)
-
+      if respond_to?(:refresh_in_provider)
```

Comment:
https://github.com/Ladas/manageiq-providers-amazon/blob/cd02dc09e805424ef90cf30f082f710000c09319/spec/models/manageiq/providers/amazon/cloud_manager/vcr_specs/targeted_refresh_spec.rb#L273-L273

I think we can integrate that to the refresh_new_target and switch based on configuration.